README.md

```diff
@@ -1,2 +1,68 @@
 # django-ilmoitin
-Django app for sending notifications
+
+[![Latest PyPI version](https://badge.fury.io/py/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+[![Python versions](https://img.shields.io/pypi/pyversions/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+
+A templated Django messaging library
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django_ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. Django ilmoitin uses Django Mailer to send emails, so you need to configure the MAILER_EMAIL_BACKEND setting, that django-mailer knows how to actually send the mail
+
+```python
+MAILER_EMAIL_BACKEND = "your.actual.EmailBackend"
+```
+
+2. Create a `notifications.py` file in django app and register your notification types:
+
+```python
+from django_ilmoitin.registry import notifications
+
+notifications.register("event_created", "Event created")
+```
+
+3. Create a `dummy_context.py` file in django app and add dummy context data:
+
+```python
+from django_ilmoitin.dummy_context import dummy_context
+
+from .models import MyModel
+
+model = MyModel(foo="bar")
+
+dummy_context.content.update({
+    "model": model
+})
+```
+
+4. Import notifications and dummy context in your apps.py:
+
+```python
+from django.apps import AppConfig
+
+
+class ExampleConfig(AppConfig):
+    name = "example"
+
+    def __init__(self, *args, **kwargs):
+        super(ExampleConfig, self).__init__(*args, **kwargs)
+        import example.notifications
+
+    def ready(self):
+        import example.dummy_context
+```
+
```

Comment:
and the 5th step is telling the user how to send emails with `django-ilmoitin` 🙂

Reply:
and also we have to mention that they need to actually create templates in django admin